azure-pipelines.yaml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docker Jenkins Images
```

Comment:
We use acr tasks for this, you don’t need a pipeline

Reply:
I've created the 2 acr tasks. here's the output for one of them:

bogdans$ az acr task create --registry $ACR_NAME --name jenkins-agent --image jenkins/agent:{{.Run.ID}} --context https://github.com/hmcts/cnp-jenkins-docker.git --file master\Dockerfile --git-access-token $GIT_PAT --subscription DCD-CNP-PROD
{- Finished ..
  "agentConfiguration": {
    "cpu": 2
  },
  "agentPoolName": null,
  "creationDate": "2020-06-18T11:03:57.818028+00:00",
  "credentials": null,
  "id": "/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/rpe-acr-prod-rg/providers/Microsoft.ContainerRegistry/registries/hmctspublic/tasks/jenkins-agent",
  "identity": null,
  "location": "uksouth",
  "name": "jenkins-agent",
  "platform": {
    "architecture": "amd64",
    "os": "linux",
    "variant": null
  },
  "provisioningState": "Succeeded",
  "resourceGroup": "rpe-acr-prod-rg",
  "status": "Enabled",
  "step": {
    "arguments": [],
    "baseImageDependencies": null,
    "contextAccessToken": null,
    "contextPath": "https://github.com/hmcts/cnp-jenkins-docker.git",
    "dockerFilePath": "masterDockerfile",
    "imageNames": [
      "jenkins/agent:{{.Run.ID}}"
    ],
    "isPushEnabled": true,
    "noCache": false,
    "target": null,
    "type": "Docker"
  },
  "tags": null,
  "timeout": 3600,
  "trigger": {
    "baseImageTrigger": {
      "baseImageTriggerType": "Runtime",
      "name": "defaultBaseimageTriggerName",
      "status": "Enabled",
      "updateTriggerEndpoint": null,
      "updateTriggerPayloadType": "Default"
    },
    "sourceTriggers": [
      {
        "name": "defaultSourceTriggerName",
        "sourceRepository": {
          "branch": "master",
          "repositoryUrl": "https://github.com/hmcts/cnp-jenkins-docker.git",
          "sourceControlAuthProperties": null,
          "sourceControlType": "Github"
        },
        "sourceTriggerEvents": [
          "commit"
        ],
        "status": "Enabled"
      }
    ],
    "timerTriggers": null
  },
  "type": "Microsoft.ContainerRegistry/registries/tasks"
}